### ISSUE TITLE
Bar isn't clickable

### ISSUE THREAD

**Issue by exiett** [FelixKratz/SketchyBar#374]
Hi! Already tried uninstalling and installing back again. Followed the tutorial all the way. Although the bar isn't "clickable". I cannot change spaces using it, I can't open my Wi-Fi settings neither other settings via the bar. What am I doing wrong? This is my `sketchybarrc`:
```
# This is a demo config to show some of the most important commands more easily.
# This is meant to be changed and configured, as it is intentionally kept sparse.
# For a more advanced configuration example see my dotfiles:
# https://github.com/FelixKratz/dotfiles

PLUGIN_DIR="$HOME/.config/sketchybar/plugins"

##### Bar Appearance #####
# Configuring the general appearance of the bar, these are only some of the
# options available. For all options see:
# https://felixkratz.github.io/SketchyBar/config/bar
# If you are looking for other colors, see the color picker:
# https://felixkratz.github.io/SketchyBar/config/tricks#color-picker

sketchybar --bar height=42        \
                 blur_radius=0   \
                 y_offset=5      \
                 position=top     \
                 sticky=off       \
                 padding_left=4  \
                 padding_right=2 \
                 shadow=on       \
                 corner_radius=12 \
                 margin=20       \
                 color=0xff1e1d2e

##### Changing Defaults #####
# We now change some default values that are applied to all further items
# For a full list of all available item properties see:
# https://felixkratz.github.io/SketchyBar/config/items

sketchybar --default icon.font="Hack Nerd Font:Bold:17.0"  \
                     icon.color=0xffffffff                 \
                     label.font="Hack Nerd Font:Bold:14.0" \
                     label.color=0xffffffff                \
                     padding_left=5                        \
                     padding_right=5                       \
                     label.padding_left=4                  \
                     label.padding_right=4                 \
                     icon.padding_left=4                   \
                     icon.padding_right=4

##### Adding Mission Control Space Indicators #####
# Now we add some mission control spaces:
# https://felixkratz.github.io/SketchyBar/config/components#space----associate-mission-control-spaces-with-an-item
# to indicate active and available mission control spaces

SPACE_ICONS=("1" "2" "3" "4" "5" "6" "7" "8" "9" "10")

for i in "${!SPACE_ICONS[@]}"
do
  sid=$(($i+1))
  sketchybar --add space space.$sid left                                 \
             --set space.$sid associated_space=$sid                      \
                              icon=${SPACE_ICONS[i]}                     \
                              background.color=0x44ffffff                \
                              background.corner_radius=5                 \
                              background.height=20                       \
                              background.drawing=off                     \
                              label.drawing=off                          \
                              script="$PLUGIN_DIR/space.sh"              \
                              click_script="yabai -m space --focus $sid"
done

##### Adding Left Items #####
# We add some regular items to the left side of the bar
# only the properties deviating from the current defaults need to be set

sketchybar --add item space_separator left                         \
           --set space_separator icon=                            \
                                 padding_left=10                   \
                                 padding_right=10                  \
                                 label.drawing=off                 \
                                                                   \
           --add item front_app left                               \
           --set front_app       script="$PLUGIN_DIR/front_app.sh" \
                                 icon.drawing=off                  \
           --subscribe front_app front_app_switched

##### Adding Right Items #####
# In the same way as the left items we can add items to the right side.
# Additional position (e.g. center) are available, see:
# https://felixkratz.github.io/SketchyBar/config/items#adding-items-to-sketchybar

# Some items refresh on a fixed cycle, e.g. the clock runs its script once
# every 10s. Other items respond to events they subscribe to, e.g. the
# volume.sh script is only executed once an actual change in system audio
# volume is registered. More info about the event system can be found here:
# https://felixkratz.github.io/SketchyBar/config/events

sketchybar --add item clock right                              \
           --set clock   update_freq=10                        \
                         icon=                                \
                         script="$PLUGIN_DIR/clock.sh"         \
                                                               \
           --add item wifi right                               \
           --set wifi    script="$PLUGIN_DIR/wifi.sh"          \
                         icon=直                               \
           --subscribe wifi wifi_change                        \
                                                               \
           --add item volume right                             \
           --set volume  script="$PLUGIN_DIR/volume.sh"        \
           --subscribe volume volume_change                    \
                                                               \
           --add item battery right                            \
           --set battery script="$PLUGIN_DIR/battery.sh"       \
                         update_freq=120                       \
           --subscribe battery system_woke power_source_change

##### Finalizing Setup #####
# The below command is only needed at the end of the initial configuration to
# force all scripts to run the first time, it should never be run in an item script.

sketchybar --update
```

**Comment by FelixKratz** [comment]
You don't seem to have registered any click actions. For the items to become clickable you need to supply a click_script, which is performed when something is clicked. The demo config does not have any of this on purpose.

**Comment by dgrebb** [comment]
@FelixKratz I came across this when I was unable to get `click_script`s working.

Unless I'm missing this, the documentation doesn't mention disabling SIP as a prerequisite for click events. It appears this is a necessary for any `click_script` and mouse events in general?

[Instructions from `yabai`](https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection) ended up working for me:

```bash
#
# APPLE SILICON
#

# If you're on Apple Silicon macOS 13.x.x
# Requires Filesystem Protections, Debugging Restrictions and NVRAM Protection to be disabled
# (printed warning can be safely ignored)
csrutil enable --without fs --without debug --without nvram
```

**Comment by FelixKratz** [comment]
@dgrebb Which OS version are you experiencing this on?

**Comment by dgrebb** [comment]
@FelixKratz `13.5.2`. I'm running good amount of your `dotfiles` plugins. But did not run `.install.sh`.

The only other change I made is disabling `yabai.sh` in `sketchybarrc`.

Looking into an issue with the `github.notification` regex now. Perhaps this locks things up for a bit? Clicks don't work on load, but do eventually register.

Will post updates. Here's the log output:

```bash
❯ sketchybar
make: `helper' is up to date.
sketchybar configuation loaded..
gh: Not Found (HTTP 404)
[2023-09-16 15:58:09] [?] Regex: No match found for regex '/github.notification\.*/'
gh: Not Found (HTTP 404)
```

Side note @FelixKratz thank you for your work on this! At a loss I only just discovered this project today.

Cheers!
